### PR TITLE
docs(CA): add docs & APIs for ChartAdvisor, Advisor, and Linter.

### PR DESCRIPTION
**Pull request by ArranZeyuWang**
### PR includes

- [ ] fixed #0
- [ ] add / modify test cases
- [ ] documents, demos

add docs & APIs for ChartAdvisor, Advisor, and Linter.

### CODE REVIEW

File: sites/ava-site/docs/api/data-wizard/statistics.en.md [hunk 1]
```diff
@@ -521,6 +421,106 @@ stats.distinct([1, 2, 3, 201, 999, 4, 5, 10]);
 // 8
 ```
 
+## minBy
+Calculate the minimum value of the specified field of the array.
+
+### Parameters
+**array** Raw array data _required_
+
+Type `[]`
```
Comment (pddpd) [inline sites/ava-site/docs/api/data-wizard/statistics.en.md:430]: Plz give a specified type, even if it's `any[]`.

File: sites/ava-site/docs/api/chart-advisor/ChartAdvisor.zh.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+title: ChartAdvisor
+order: 1
+---
+
+`markdown:docs/common/style.md`
+
+<div class='doc-md'>
+
+```sign
+Class ChartAdvisor(config?: Partial<Pick<ChartAdvisor, 'ckbCfg' | 'ruleCfg'>> = {})
```
Comment (chenluli) [inline sites/ava-site/docs/api/chart-advisor/ChartAdvisor.zh.md:11]: Should the differences between ChartAdvisor and Advisor be explained? ChartAdvisor combines Advisor and Linter, users may not be aware of this difference.

Comment (ArranZeyuWang) [reply sites/ava-site/docs/api/chart-advisor/ChartAdvisor.zh.md:11]: The difference was explained in guide/chart-advisor/ChartAdvisor. Should the API files contain such explanations? cc @neoddish

**Review by neoddish — APPROVED** [review]
LG

**Pull request merged**